packages/dart_frog_gen/example/main.dart

```diff
@@ -0,0 +1,10 @@
+// ignore_for_file: avoid_print
+
+import 'dart:io';
+
+import 'package:dart_frog_gen/dart_frog_gen.dart';
+
+void main() {
+  final routeConfiguration = buildRouteConfiguration(Directory.current);
+  print(routeConfiguration);
```

Comment:
Instead of print we could write to stout or do a developer log?

Reply:
yeah I thought about it but developer logs don't appear in the console afaik. Happy to change it though 👍

Reply:
Updated to use stdout